packages/conf-python-3-7/conf-python-3-7.1.0.0/opam

```diff
@@ -11,25 +11,27 @@ depends: [
 depexts: [
   ["python3.7"]
     {os-distribution = "ubuntu" & os-version < "20.04"}
-  ["python3"] {os-family = "debian"
+  ["python3"] {(os-family = "debian" | os-family = "ubuntu")
     & !(os-distribution = "ubuntu" & os-version < "20.04")}
   ["python3"] {os-distribution = "nixos"}
   ["python3"] {os-distribution = "alpine"}
   ["python37" "epel-release"] {os-distribution = "centos"}
   ["python3"] {os-distribution = "fedora"}
-  ["python38"] {os-distribution = "ol"} # No python37 package
+  ["python3"] {os-distribution = "ol" & os-version >= "9"}
+  ["python38"] {os-distribution = "ol" & os-version < "9"} # No python37 package
```

Comment:
This is also not really python 3.7, but I guess this will install other versions in most other systems anyway

Reply:
Note: the package description says
 `This package can only install if a Python interpreter >=3.7 is available on the system.`
so as long as it is greater that 3.7 we should be good.

I checked that Oracle 9 installs 3.9 https://yum.oracle.com/repo/OracleLinux/OL9/baseos/latest/x86_64/index.html
Finally, Oracle is now gone from opam-repo-ci :shrug: